docs_app/tools/transforms/angular-api-package/index.js

```diff
@@ -87,15 +90,15 @@ module.exports = new Package('angular-api', [basePackage, typeScriptPackage])
 
   // Additional jsdoc config (for RxJS source)
   .config(function(parseTagsProcessor) {
-    parseTagsProcessor.tagDefinitions.push({ name: 'internal' });
```

Comment:
Why did you removed the internal tag? I think this one is used to hide internal api which we will need in future?

Reply:
I though that when something is marked as `@internal` it shouldn't be part of the public API exported by neither `index.ts` nor `operators/index.ts` so if I remove this tag definition it's going to throw warnings  about an unknown tag when running `npm run docs` and thus telling me that I'm exporting something that's supposed to be internal.

Reply:
Ah ok good point. Let's stick to your approach. We have to think about it again when we start hiding the internal api.

Reply:
We use the internal flag in Angular docs to indicate that a member of a public class is not to be exposed in the docs. Also `@internal` is used by some tooling (TypeScript??) I think. So unless you do not expect this tag to appear in your source at all, I would leave it in.

Reply:
I'm checking it again and the `@internal` tag is actually already included:

https://github.com/ReactiveX/rxjs/blob/master/docs_app/tools/transforms/angular-api-package/tag-defs/internal.js

https://github.com/ReactiveX/rxjs/blob/master/docs_app/tools/transforms/angular-api-package/index.js#L81-L86

So there no need to add it again with `parseTagsProcessor.tagDefinitions.push({ name: 'internal' });`